Transfer/Sources/TransferAllFundsCell.swift

```diff
@@ -85,7 +85,8 @@ final class TransferAllFundsCell: UITableViewCell {
                    currencyCode: String?) {
         guard let availableBalance = availableBalance,
             let currencyCode = currencyCode else {
-                availableFundsLabel.text = ""
+                availableFundsLabel.text = "Amount available: N/A"
```

Comment:
This should come from localization

Reply:
Yes added translation for English. Code updated.Waiting for other language translations.